README.md

```diff
@@ -58,6 +58,8 @@ In order to deploy both the stacks the user needs to provide a set of required a
 | account (Optional)                | string  | User provided aws account                                                                                                                                                                                                                                                                        |
 | dataNodeStorage (Optional)        | string  | User provided ebs block storage size, defaults to 100Gb                                                                                                                                                                                                                                          |
 | mlNodeStorage (Optional)          | string  | User provided ebs block storage size, defaults to 100Gb                                                                                                                                                                                                                                          |
+| heapSize (Optional)               | string  | User provided JVM heap memory size (integer) in Gb, defaults to 1Gb, e.g. `--context heapSize=2` to set heap to 2Gb                                                                                                                                                                              |
```

Comment:
I think better to keep the naming same as property `jvmHeapSize`. Also shouldn't this be an integer?
```suggestion
| jvmHeapSize (Optional)               | integer  | User provided JVM heap memory size (integer) in Gb, defaults to 1Gb, e.g. `--context heapSize=2` to set heap to 2Gb                                                                                                                                                                              |
```